gulp/util/webpack-config.js

```diff
@@ -56,7 +56,6 @@ const EXTERNALS = {
     "jquery": "$",
     "moment": "moment",
     "react": "React",
-    "react-addons-css-transition-group": "React.addons.CSSTransitionGroup",
```

Comment:
Hmm, I'm getting `Cannot find module "react-transition-group/CSSTransitionGroup"` on the built artifact - maybe this should be something else? I'm not too familiar with webpack externals!